CHANGELOG.md

```diff
@@ -10,4 +10,58 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [0.1.0] - 2021-09-03
 ### Added
-- Initial implementation
\ No newline at end of file
+- Initial implementation
+
+## [1.1.0] - 7/21/2022
+### Fixed
+### Added
```

Comment:
Thanks for adding in this history! I think this looks good and makes sense, but I just wanted to mention that going forward each feature/bug fix should be associated with a GitHub issue and each entry in the changelog should just be a reference to the issue as described in CONVENTIONS.md.